R/class-yarn.R

```diff
@@ -199,6 +199,32 @@ yarn <- R6::R6Class("yarn",
         message("to use the `protect_unescaped()` method, you will need to re-read your document with `yarn$new(sourcepos = TRUE)`")
       }
       invisible(self)
+    },
+    #' @description Return nodes whose contents are protected from being escaped
+    #' @param type a character vector listing the protections to be included.
+    #'   Defaults to `NULL`, which includes all protected nodes:
+    #'   - math: via the `protect_math()` function
```

Comment:
```suggestion
    #'   - math: via the [protect_math()] function
```

Referring to https://roxygen2.r-lib.org/articles/rd-formatting.html#function-links as I always forget how it works :sweat_smile: